diesel_derives/src/lib.rs

```diff
@@ -237,6 +237,12 @@ pub fn derive_identifiable(input: TokenStream) -> TokenStream {
 /// example, an enum that maps to a label and a value column). Add
 /// `#[diesel(embed)]` to any such fields.
 ///
+/// To provide custom serialization behavior for a field, you can use
+/// `#[diesel(serialize_as = "SomeType")]`. If this attribute is present, Diesel
+/// will call `.into` on the corresponding field and serialize the instance of `SomeType`,
```

Comment:
We are calling `from` not `into`. Either change the implementation or the documentation. (I would say change the implementation as calling `into` is generally preferred as it is more generically)

Reply:
Yes, I started with the documentation but then ran into problems with `.into` because rustc couldn't resolve the types for cases where `serialize_as` is **not** used. Will update the documentation in the end!

Reply:
You probably need to use UFCS to specify all generic parameters there.